event-handler/event_handler_test.go

```diff
@@ -221,13 +227,24 @@ func (s *EventHandlerSuite) TestEvents() {
 	stdinPipe, err := cmd.StdinPipe()
 	require.NoError(t, err)
 
+	cmdStdout := &bytes.Buffer{}
+	cmdStderr := &bytes.Buffer{}
+
+	cmd.Stdout = cmdStdout
+	cmd.Stderr = cmdStderr
+
 	err = cmd.Start()
 	require.NoError(t, err)
 
-	_, err = stdinPipe.Write([]byte("exit\n"))
+	_, err = stdinPipe.Write([]byte("whoami\n\r"))
```

Comment:
Calling `exit` immediately caused the process to fail
Maybe it was trying to setup the shell :question: